src/components/GloballySearched.js

```diff
@@ -0,0 +1,72 @@
+import Carousel from "react-material-ui-carousel";
+import { Paper } from "@mui/material";
+import MusicPlayer from "./MusicPlayer";
+import sampleSongs from "../sample/sample";
+
+export default function GloballySearched({ songsPerGroup }) {
+  var songs = sampleSongs.tracks.items;
+
+  const nGroups = Math.floor(songs.length / 3);
+
+  const songGroups = [];
+
+  for (let i = 0; i < nGroups; i++) {
+    songGroups.push([]);
+    for (let j = i * songsPerGroup; j < (i + 1) * songsPerGroup; j++) {
+      if (j < songs.length) {
+        songGroups[i].push(songs[j]);
+      }
+    }
+  }
+
+  return (
+    <div className='carousel-container'>
+      <h3 className='carousel-title'>Songs Searched for Globally</h3>
+      <div id='carousel'>
+        <Carousel
+          animation='slide'
+          indicators={false}
+          duration={500}
+          autoPlay={true}
+        >
+          {songGroups.map((group, i) => {
+            return <SongGroup key={i} songs={group} />;
+          })}
+          {}
+        </Carousel>
+      </div>
+    </div>
+  );
+}
+
+function SongGroup({ songs }) {
```

Comment:
Should this be its own component?

Reply:
Made it isolated as we likely won't be reusing this type of component. It's specific to the carousel component.

Reply:
Ok that makes sense. Thanks for clarifying!